package-lock.json

```diff
@@ -11,6 +11,7 @@
 			"dependencies": {
 				"@types/js-cookie": "^3.0.6",
 				"@types/react-router-dom": "^5.3.3",
+				"@wordpress/date": "^5.11.0",
```

Comment:
If we look on the left of this `package-lock.json` diff, there are `"dev": true,` entries that have been removed. This is what my question was about. I'm guessing this was done by the `@wordpress/date` addition, though I can't explain why it happens to seemingly unrelated packages.

Maybe there's no good way to answer this question.

Reply:
Ah! Sorry, I was looking at the `package.json` file 🤦  I think this is because I installed the `@wordpress/date` as a dependency where it should be a dev dependency. I will tweak this - and also move any other instances of dev dependencies that are mistakenly installed as dependencies.